features/webdriver-sensors.yml

```
@@ -1,5 +1,4 @@
-draft_date: 2024-08-27
-name: WebDriver Sensors
+name: WebDriver sensors
 description: WebDriver Sensors are WebDriver extension commands for creating, deleting, getting, and setting readings for virtual sensors (sensors that do not depend on underlying hardware or operating system support).
```

```suggestion
name: Virtual sensors
description:  The `CreateVirtualSensor` WebDriver command creates a virtual sensor for testing applications that use sensor APIs, without the need for real device sensors.
```

It's be nice to rename the file too.